1회차/이상욱/4_창용마을무리의개수(런타임에러).py

```diff
@@ -0,0 +1,43 @@
+import sys
+sys.stdin = open("_창용마을무리의개수.txt")
+
+# def dfs(start):
+#     stack = [start]
+#     visited[start] == True
+
+#     while stack:
+#         cur = stack.pop()
+#         for adj in con[cur]:
+#             if not visited[adj]:
+#                 visited[adj] = True
+#                 stack.append(adj)
+
+T = int(input())
+for t in range(1, T+1):
+    n, m = map(int, input().split())
+    con = [[] for _ in range(n+1)]
+    for _ in range(m):
+        N, M = map(int, input().split())
+        con[N].append(M)
+        con[M].append(N)
+
+    visited = [False] * (n+1)
+    cnt = 0
+
+    for i in range(1, N+1):
```

Comment:
변수끼리 헷갈리셨던 것 같아요! 해당 부분만 바꾸면 정답 코드입니다.
```suggestion
    for i in range(1, n+1):
```